_posts/2016-08-09-collections-app.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Collecting Connections through the Powerbox"
+author: David Renshaw
+authorUrl: https://github.com/dwrensha
+---
+
+Starting work on a project in Sandstorm often means creating many grains, each pertaining to a
+different aspect of your work. In a typical project, you might have a spreadsheet, a chat room, a
+kanban board, and several source code repositories. Sandstorm makes it easy to share these grains
+with your collaborators, but until recently, you would need to share each grain to each collaborator
+separately --- a task that could quickly become tedious. What if you want to share all of the
+project's grains as a single unit?
+
+Now that we have released the
```

Comment:
I know you say here it's an "app", but I honestly figured it was some deep core system plugin thing, not the same kind of "app" anyone can write, even though I know it's not, just because the idea is so strange. I noticed this because when I got to the section where you mention that it's an app, I was surprised that it was an app.

You don't necessarily have to do anything about this reaction of mine.